### PR TITLE
Enable draft installer

### PR DESCRIPTION
**Pull request by Tatsinnit**
# Description

This PR enables the `DRAFT CLI Installer` which means user no longer need to go download the cli binary based on their operating system, that all can be taken care of by this.

This enhancement will also take care of https://github.com/Azure/draft/issues/143 which asks for `JQ processor installation`. 🎊🚀🥷

As we develop: this just need minor readme update. like cli location will be `usr/local/bin/...` and its easily callable. cc: @qpetraroia thanks. (Also for windows user Installation a note like various other tools that - `Windows isn't currently supported (you can use WSL)`)

Thank you so much all. ❤️☕️🚀 **cc**: @bfoley13, @hsubramanianaks , @davidgamero , @qpetraroia, @sabbour  **fyi**:  @gambtho 

**Simple usage will looks like:**

```*.sh
# usage: 
    curl -fsSL https://raw.githubusercontent.com/Tatsinnit/draft/feature/draft-installer/scripts/install.sh | sh
```

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

- [x] I have tested this on my Mac, ~will test in for wsl today~ Now tested on windows which should be sweet with ubuntu intern. (Screenshots below)

Screen Shot 2022-10-02 at 8 59 46 PM

MicrosoftTeams-image (2)

MicrosoftTeams-image (3)

### CODE REVIEW

**Review by hsubramanianaks — APPROVED** [review]
Looks good to me

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  log INFO "Checking locally installed JQ Processor version"
+  jqversion=$(jq --version)
+  log INFO "Locally installed JQ Processor version is $jqversion"
+  if [ -z "${jqversion}" ]; then
+    $PACKAGER install jq
+  fi
+  jqversion=$(jq --version)
+  set -e
+  if [ -z "${jqversion}" ]; then
+    echo "Your machine donot have JQ processor installed, plaese make sure JQ Processor is installed. \n please perfor $PACKAGER update and retry running scripts"
```
Comment (davidgamero) [inline scripts/install.sh:101]: nit: typo on `perfor`

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+# ######                                 #####  #       ### 
+# #     # #####    ##   ###### #####    #     # #        #  
+# #     # #    #  #  #  #        #      #       #        #  
+# #     # #    # #    # #####    #      #       #        #  
+# #     # #####  ###### #        #      #       #        #  
+# #     # #   #  #    # #        #      #     # #        #  
+# ######  #    # #    # #        #       #####  ####### ###    
+#                                                               
+# usage: 
+#    curl -fsSL https://raw.githubusercontent.com/Tatsinnit/draft/feature/draft-installer/scripts/install.sh | sh
```
Comment (davidgamero) [inline scripts/install.sh:13]: prior to merging this PR we should update this URL to reflect the Azure/draft repo

**Review by hsubramanianaks — APPROVED** [review]
Looks good

**Review by hsubramanianaks — APPROVED** [review]
Looks good to me, tested this in Windows here is the screenshot 
image

**Comment by davidgamero** [comment]
lgtm

**Comment by davidgamero** [comment]
@Tatsinnit it appears one of your commits isn't signed, you'll need to sign that one for this to be able to merge

**Comment by Tatsinnit** [comment]
> @Tatsinnit it appears one of your commits isn't signed, you'll need to sign that one for this to be able to merge

Thanks for this, done, and pushed all commits are verified now. thanks. 🎊

**Review by hsubramanianaks — APPROVED** [review]
looks good to me.

**Pull request merged**